### PR TITLE
Fixed the E2E application test

### PR DESCRIPTION
**Pull request by yanmxa**
Signed-off-by: myan [myan@redhat.com](mailto:myan@redhat.com)
Reference: https://github.com/stolostron/backlog/issues/24506

### CODE REVIEW

File: cicd-scripts/run-prow-e2e-test.sh [hunk 1]
```diff
@@ -18,10 +18,11 @@ echo "export OPENSHIFT_CI=$OPENSHIFT_CI" >> ${ROOT_DIR}/test/resources/env.list
 echo "export LOG=/dev/stdout" >> ${ROOT_DIR}/test/resources/env.list
 echo "export LEAF_HUB_LOG=/dev/stdout" >> ${ROOT_DIR}/test/resources/env.list
 echo "export VERBOSE=6" >> ${ROOT_DIR}/test/resources/env.list
+echo "export HUB_OF_HUBS_MANAGER_IMAGE_REF=quay.io/myan/hub-of-hubs-manager:latest" >> ${ROOT_DIR}/test/resources/env.list
```
Comment (morvencao) [inline cicd-scripts/run-prow-e2e-test.sh:21]: Is this intent? you're using your dev image

Comment (yanmxa) [reply cicd-scripts/run-prow-e2e-test.sh:21]: I'm using the dev image for now. And I'm not sure whether we should replace the image "quay.io/open-cluster-management-hub-of-hubs/hub-of-hubs-manager" with this latest code?

File: manager/deploy/operator.yaml.template [hunk 2]
```diff
@@ -23,8 +23,7 @@ spec:
             - --manager-namespace=$(POD_NAMESPACE)
             - --watch-namespace=$(WATCH_NAMESPACE)
             - --transport-type=${TRANSPORT_TYPE}
-            - --process-database-url=$(PROCESS_DATABASE_URL)
-            - --transport-bridge-database-url=$(TRANSPORT_BRIDGE_DATABASE_URL)
+            - --admin-database-url=$(ADMIN_DATABASE_URL)
```
Comment (morvencao) [inline manager/deploy/operator.yaml.template:26]: can you also change the envs `PROCESS_DATABASE_URL` and `TRANSPORT_BRIDGE_DATABASE_URL` to `ADMIN_DATABASE_URL` in the `env` section?

**Comment by morvencao** [comment]
As we discussed in slack, we agreed to move DB initialization work to operator.

**Review by morvencao — COMMENTED** [review]
/lgtm

**Pull request merged**